src/plugins/tjvendors/tjvendors/language/en-GB/en-GB.plg_tjvendors_tjvendors.sys.ini

```diff
@@ -0,0 +1,2 @@
+PLG_TJVENDORS_XML_DESCRIPTION="TJVendors Plugin"
+PLG_TJVENDORS="TJVendors plugin"
```

Comment:
i think these needs to have prefixes as

PLG_TJVENDORS_TJVENDORS

plg_type_name

Reply:
will check and update while testing